source/_integrations/lametric.markdown

```diff
@@ -1,10 +1,9 @@
 ---
 title: LaMetric
-description: Instructions on how to integrate LaMetric with Home Assistant.
+description: Instructions on how to integrate LaMetric TIME with Home Assistant.
 ha_category:
-  - Hub
   - Notifications
-ha_iot_class: Cloud Push
+ha_iot_class: Local Polling
```

Comment:
The core integration manifest says local_push. Should that change too?

Reply:
Yeah it will change back in a next PR... currently it does only notifications, so it is push. But the next PR introduces the coordinator, device and entities.

But at this point, you are right 👍 

```suggestion
ha_iot_class: Local Push
```